kobo/apps/openrosa/apps/api/tests/viewsets/test_xform_submission_api.py

```diff
@@ -458,6 +461,42 @@ def test_edit_submission_with_service_account(self):
                 response['Location'], 'http://testserver/submission'
             )
 
+    def test_submission_account_inactive(self):
```

Comment:
Sorry this is so long :neutral_face: I copied `test_submission_blocking_flag()` and did slim it down somewhat